tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -422,6 +422,55 @@ public void TestBinaryOperationsOnBoolColumn()
             Assert.Equal(true, newdf1[4, 1]);
         }
 
+        [Fact]
+        public void TestBinaryOperationsOnArrowStringColumn()
+        {
+            var df = new DataFrame();
+            var strArrayBuilder = new StringArray.Builder();
+            for (int i = 0; i < 10; i++)
+            {
+                strArrayBuilder.Append(i.ToString());
+            }
+            StringArray strArray = strArrayBuilder.Build();
+
+            ArrowStringDataFrameColumn stringColumn = new ArrowStringDataFrameColumn("String", strArray.ValueBuffer.Memory, strArray.ValueOffsetsBuffer.Memory, strArray.NullBitmapBuffer.Memory, strArray.Length, strArray.NullCount);
+            df.Columns.Insert(0, stringColumn);
+
+            DataFrameColumn newCol = stringColumn.ElementwiseEquals(4);
+            Assert.Equal(true, newCol[4]);
+            Assert.Equal(false, newCol[0]);
+            Assert.Equal(false, newCol[5]);
+
+            newCol = (stringColumn as ArrowStringDataFrameColumn).ElementwiseEquals("4");
```

Comment:
`stringColumn` is already casted as a `ArrowStringDataFrameColumn`. Why the extra cast here?